### PR TITLE
fix/SMC-3689

### PR DESCRIPTION
**Pull request by southbite**

### CODE REVIEW

File: test/_lib/integration-02-local-component1/package.json [hunk 1]
```diff
@@ -13,10 +13,7 @@
         },
         "remoteComponent3": {
           "version": "^2.0.0",
-          "methods": {
-            "method1": {},
-            "method2": {}
-          }
+          "discoverMethods": true
```
Comment (csampsonza) [inline test/_lib/integration-02-local-component1/package.json:16]: This looks new, probably need to add it to the documentation

File: test/integration/19-persist-members-restarts.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ var Promise = require('bluebird');
 const clearMongoCollection = require('../_lib/clear-mongo-collection');
 const testHelper = require('../_lib/test-helper');
 
-describe(testHelper.testName(__filename, 3), function() {
+xdescribe(testHelper.testName(__filename, 3), function() {
```
Comment (csampsonza) [inline test/integration/19-persist-members-restarts.js:7]: Typo?

File: test/_lib/integration-02-local-component1/package.json (hunk 1 above)
Comment (southbite) [reply test/_lib/integration-02-local-component1/package.json:16]: Absolutely - but I just want to write some more tests in happner-client

File: test/integration/19-persist-members-restarts.js (hunk 2 above)
Comment (southbite) [reply test/integration/19-persist-members-restarts.js:7]: No - this is functionality we want to remove in future - which creates intermittent test failures , SMC does not use the persistMembers stuff anymore

**Pull request merged**